extensions/api/data-management/policydefinition/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/policy/service/PolicyService.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022 ZF Friedrichshafen AG
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributors:
+ *    ZF Friedrichshafen AG - Initial API and Implementation
+ */
+
+package org.eclipse.dataspaceconnector.api.datamanagement.policy.service;
+
+import org.eclipse.dataspaceconnector.api.result.ServiceResult;
+import org.eclipse.dataspaceconnector.policy.model.Policy;
+import org.eclipse.dataspaceconnector.spi.query.QuerySpec;
+
+import java.util.Collection;
+
+public interface PolicyService {
+
+    /**
+     * Returns a policy by its id
+     *
+     * @param policyId id of the policy
+     * @return the policy, null if it's not found
+     */
+    Policy findById(String policyId);
```

Comment:
add `@NotNull` and `@Nullable` annotations to methods on interface and implementation?